### PR TITLE
Solved some liting issues recognized by SonarLin

### PR DESCRIPTION
**Pull request by ChrisRo89**

### CODE REVIEW

File: src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java [hunk 1]
```diff
@@ -72,12 +71,7 @@
 import java.net.URISyntaxException;
 import java.net.UnknownHostException;
 import java.security.MessageDigest;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Hashtable;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (jglick) [inline src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java:74]: :-1:

File: src/main/java/jenkins/security/plugins/ldap/LDAPGroupMembershipStrategy.java [hunk 2]
```diff
@@ -78,7 +79,7 @@ public void setAuthoritiesPopulator(LdapAuthoritiesPopulator authoritiesPopulato
      * @throws DataAccessException if there is an error performing the search.
      * @since 1.18
      */
-    public Set<String> getGroupMembers(String groupDn, LDAPConfiguration conf) throws DataAccessException {
-        return null;
+    public Set<String> getGroupMembers(String groupDn, LDAPConfiguration conf) {
+        return Collections.emptySet();
```
Comment (jglick) [inline src/main/java/jenkins/security/plugins/ldap/LDAPGroupMembershipStrategy.java:83]: Contradicts the Javadoc.

File: src/main/java/hudson/security/DelegateLdapUserDetails.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+import org.acegisecurity.GrantedAuthority;
+import org.acegisecurity.userdetails.ldap.LdapUserDetails;
+
+import javax.annotation.Nonnull;
+import javax.naming.directory.Attributes;
+import javax.naming.ldap.Control;
+import java.io.Serializable;
+
+/*package access for testability*/
+class DelegatedLdapUserDetails implements LdapUserDetails, Serializable {
```
Comment (jglick) [inline src/main/java/hudson/security/DelegateLdapUserDetails.java:13]: Duplicating code? Why?

**Comment by jglick** [comment]
I have major refactorings to this repository in flight, so I would not encourage cosmetic PRs at this time. Probably best closed.

**Comment by ChrisRo89** [comment]
Ok, all right

**Pull request closed without merging**